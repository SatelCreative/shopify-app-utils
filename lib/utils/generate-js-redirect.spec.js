const generateShopifyRedirect = require('./generate-js-redirect');

describe('Generate Shopify Redirect', () => {
  it('should generate a javascript based redirect', () => {
    expect(
      generateShopifyRedirect({ url: 'https://example.com' }),
    ).toMatchSnapshot();
  });
});
