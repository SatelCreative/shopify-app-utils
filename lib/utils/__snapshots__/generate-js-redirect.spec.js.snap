// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Generate Shopify Redirect should generate a javascript based redirect 1`] = `"<!DOCTYPE html><html><head><script type=\\"text/javascript\\">window.top.location.href = \\"https://example.com\\"</script></head></html>"`;
